autoPyTorch/pipeline/components/setup/network/base_network.py

```diff
@@ -20,15 +20,11 @@ class NetworkComponent(autoPyTorchTrainingComponent):
 
     def __init__(
             self,
-            network: Optional[torch.nn.Module] = None,
```

Comment:
I think we should keep the network here.
So what happens with base estimators is, that when clone, they honor the set_params/get_params scheme from scikit learn described [here](https://scikit-learn.org/stable/developers/develop.html). In summary, an object gets re-created using the class-type and whatever get_params returns. Get_params returns the contents of `vars` with some filtering.

Reply:
Okay, I will add it again